[2024-12-15T23:49:18.972+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data manual__2024-12-15T23:49:12+00:00 [queued]>
[2024-12-15T23:49:18.982+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data manual__2024-12-15T23:49:12+00:00 [queued]>
[2024-12-15T23:49:18.983+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-15T23:49:18.997+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_data> on 2024-12-15 23:49:12+00:00
[2024-12-15T23:49:19.001+0000] {standard_task_runner.py:57} INFO - Started process 362 to run task
[2024-12-15T23:49:19.005+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'store_data', 'manual__2024-12-15T23:49:12+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmp524bn_l2']
[2024-12-15T23:49:19.009+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask store_data
[2024-12-15T23:49:19.079+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.store_data manual__2024-12-15T23:49:12+00:00 [running]> on host 6cd16bdff3c4
[2024-12-15T23:49:19.207+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='store_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T23:49:12+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-15T23:49:12+00:00'
[2024-12-15T23:49:19.251+0000] {data_warehouse.py:50} INFO - Database tables initialized successfully
[2024-12-15T23:49:19.251+0000] {data_warehouse.py:66} INFO - Saving data to warehouse table: covid_daily_stats
[2024-12-15T23:49:19.291+0000] {data_warehouse.py:92} INFO - Successfully saved 231 records to covid_daily_stats
[2024-12-15T23:49:19.292+0000] {python.py:194} INFO - Done. Returned value was: Data stored successfully
[2024-12-15T23:49:19.330+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=store_data, execution_date=20241215T234912, start_date=20241215T234918, end_date=20241215T234919
[2024-12-15T23:49:19.352+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-15T23:49:19.383+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
